### DISCUSSION TITLE
How to import framer-motion dynamically using nextjs version 13.5.6?

### DISCUSSION THREAD

**Discussion by MkMonir** [vercel/next.js#70700] [App Router] [answered]
const {motion}= dynamic(() =>
  import("framer-motion"))
);

**Comment by hasan-mohammad-bd** [comment, marked as answer]
const MotionDiv = dynamic(() =>
  import("framer-motion").then((mod) => mod.motion.div)
);
Please try this one

**Reply by MkMonir** [reply]
Thanks. It worked.
